content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -154,7 +153,239 @@ Set the value to `true` to disable data export, e.g. for testing purposes.
+otel.resource.attributes.xyz=foo
+```
+
+It's also possible to specify the resource attributes in `application.yaml`:
+
+```yaml
+otel:
+  resource:
+    attributes:
+      environment: dev
+      xyz: foo
+```
+
+Finally, the resource attributes can be specified as a comma-separated list, as
+described in the
+[specification](/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_resource_attributes):
+
+```shell
+export OTEL_RESOURCE_ATTRIBUTES="key1=value1,key2=value2"
+```
+
+The service name is determined by the following precedence rules, in accordance
+with the OpenTelemetry
+[specification](/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_service_name):
+
+1. `otel.service.name` spring property or `OTEL_SERVICE_NAME` environment
+   variable (highest precedence)
+2. `service.name` in `otel.resource.attributes` system/spring property or
+   `OTEL_RESOURCE_ATTRIBUTES` environment variable
+3. `service.name` in `otel.springboot.resource.attributes` system/spring
+   property
+4. `spring.application.name` spring property
+5. The default value is `unknown_service:java` (lowest precedence)
+
+### Automatic instrumentation
+
+Auto-configures OpenTelemetry instrumentation for
+[spring-web](#spring-web-auto-configuration) ,
+[spring-webmvc](#spring-web-mvc-auto-configuration), and
+[spring-webflux](#spring-webflux-auto-configuration). Leverages Spring Aspect
+Oriented Programming, dependency injection, and bean post-processing to trace
+spring applications.
+
+| Feature        | Property                                      | Default Value | ConditionalOnClass     |
+| -------------- | --------------------------------------------- | ------------- | ---------------------- |
+| spring-web     | `otel.instrumentation.spring-webmvc.enabled`  | true          | `RestTemplate`         |
+| spring-webmvc  | `otel.instrumentation.spring-web.enabled`     | true          | `OncePerRequestFilter` |
+| spring-webflux | `otel.instrumentation.spring-webflux.enabled` | true          | `WebClient`            |
+
+#### Spring Web Auto Configuration
+
+Provides auto-configuration for the OpenTelemetry RestTemplate trace interceptor
+defined in
+[opentelemetry-spring-web-3.1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/spring-web/spring-web-3.1).
+This auto-configuration instruments all requests sent using Spring RestTemplate
+beans by applying a RestTemplate bean post processor. This feature is supported
+for spring web versions 3.1+.
+[Spring Web - RestTemplate Client Span](#example-trace---resttemplate-client-span)
+show cases a sample client span generated by this auto-configuration. Check out
+[opentelemetry-spring-web-3.1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/spring-web/spring-web-3.1)
+to learn more about the OpenTelemetry RestTemplateInterceptor.
+
+##### Example Trace - RestTemplate Client Span
```

Comment:
the example traces are pretty old / outdated. we could update them all, but wonder if we want to keep them? I don't think we typically include example traces elsewhere in docs(?)

Reply:
I agree to remove these examples